types/html-minifier/v3/html-minifier-tests.ts

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
@rikuayanokozy looks like you did the opposite of what @minestarks intended. Generally you would move the old version to a 'Vx' folder, and make the new version the default. So in your case, you would move what's now in DT to 'v1', and keep your changes in the root.

Reply:
🤣OMG, What am I doing??!
A new commit fix that, please check it.